numpyro/contrib/module.py

```diff
@@ -223,7 +223,12 @@ def _update_params(params, new_params, prior, prefix=""):
             new_item = new_params[name]
             _update_params(item, new_item, prior, prefix=flatten_name)
         elif (not isinstance(prior, dict)) or flatten_name in prior:
-            d = prior[flatten_name] if isinstance(prior, dict) else prior
```

Comment:
Could you update docstring of `random_flax_module` and `random_haiku_module` to reflect this new feature?